docs/software/home_automation.md

```diff
@@ -81,85 +81,85 @@ Home Assistant is an open-source home automation platform running on Python 3. T
 
 Official documentation: <https://home-assistant.io/docs>
 
-## EmonPi
+## emonHub
 
-Turn your Raspberry Pi into a energy usage monitor with web interface.
+Turn your Raspberry Pi into an energy usage monitor with web interface.
 
-![EmonPi wen interface screenshot](../assets/images/dietpi-software-homeautomation-emonpi.png){: width="400" height="237" loading="lazy"}
+![emonCMS web interface screenshot](../assets/images/dietpi-software-homeautomation-emonpi.png){: width="400" height="237" loading="lazy"}
 
 ### Installation
 
-The DietPi optimized installation for EmonPi is aimed at users who want the following:
+The DietPi optimised installation for emonPi is aimed at users who want the following:
 
-- Ultra lightweight alternative installation to the official EmonPi image, with all the optimisations and features of DietPi. Allowing additional uses for your RPi device (e.g.: ownCloud server): [htop image of RPi Zero @700 MHz](https://dietpi.com/downloads/misc/EmonPi_Guide/EmonPi_DietPi_zero_700mhz.jpg), running our EmonPi installation.
+- Ultra lightweight alternative installation to the official emonPi image, with all the optimisations and features of DietPi. Allowing additional uses for your RPi device (e.g.: ownCloud server): [htop image of RPi Zero @700 MHz](https://dietpi.com/downloads/misc/EmonPi_Guide/EmonPi_DietPi_zero_700mhz.jpg), running our emonHub installation.
 - Real time and historical energy usage statistics uploaded to the [emoncms.org](https://emoncms.org/) cloud, viewable from a web browser and the [Android App](https://openenergymonitor.org/forum-archive/node/11260.html), from anywhere in the world.
-- Use an existing RPi, or, do not wish to purchase the full EmonPi package (e.g.: case).
-- Comfortable attaching the EmonPi to your Raspberry Pi GPIO. No soldering or wiring is required.
+- Use an existing RPi, or, do not wish to purchase the full emonPi package with case and LCD.
```

Comment:
Proposal: -> Use an existing RPi or purchase the full emonPi package with case and LCD.

Reply:
Note the **not**. The full emonPi package contains the RPi with case and LCD and a ready-to-run disk image, from what I understand. Also likely DietPi does not support this particular LCD OOTB, at least it has not been tested. So our implementation is then indeed addressing users who have an RPi already and/or want to **not** purchase the full package, but a light solution with a custom RPi model (not sure which one is included in the full package) and the raw/bulk emonPi shield (£23.82 vs min £123.59 for full package).

Reply:
Shall we rephrase the sentence to make this clearer?